vllm/config.py

```diff
@@ -137,6 +137,7 @@ def __init__(
         skip_tokenizer_init: bool = False,
         served_model_name: Optional[Union[str, List[str]]] = None,
         limit_mm_per_prompt: Optional[Mapping[str, int]] = None,
+        use_output_proc_callback: Optional[bool] = True,
```

Comment:
+1

Reply:
Same comment as before: Since we implemented the overlap of `process_model_outputs` and GPU forward pass using a callback function and its not really "asynchronous", I think `use_output_proc_callback` is a better choice.

Reply:
Don't have any strong opinions though. If you think "async" is a better wording, I'll change it everywhere

Reply:
My feeling is the name "callback" should take a function, so it's a bit confusing from a user perspective when specifying "use_callback=True". After all the term "callback" itself doesn't explain the real functionality

Reply:
Changed everywhere to use_async_output_proc